<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSourceDialog</class>
 <widget class="QDialog" name="DataSourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Source Setting</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QRadioButton{

font-weight: bold;
font-size: 15px;

}


QFrame{
border: 3px solid gray;
border-radius: 40px;
background: white;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}
QLabel{
border: none;
background: white;
font-weight: bold;
font-size: 15px;

}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="radioButton_remoteSource">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>13</y>
      <width>185</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Source from remote</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_apply">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_local_source_seq">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>45</y>
      <width>248</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Source from local seq Dir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_set_local_file">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Setting Video file</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_local_file_source">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>29</y>
      <width>275</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Source from local video file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_set_local_seq">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Setiing Dir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
